system_setup/ui/views/wslconfadvanced.py

```diff
@@ -128,15 +135,24 @@ def __init__(self, controller, configuration_data):
 
     def done(self, result):
         self.controller.done(WSLConfigurationAdvanced(
```

Comment:
I think the stuttering here can be fixed by building a dict from the self.form values. I suggest you do that in another PR, as this is not related to this one.

Reply:
Gotcha